src/scr.c

```diff
@@ -2417,11 +2435,13 @@ int SCR_Finalize()
 #endif /* HAVE_LIBPMIX */
 
   /* shut down the AXL library */
-  int axl_rc = AXL_Finalize_comm(scr_comm_world);
-  if (axl_rc != AXL_SUCCESS) {
-    scr_abort(-1, "Failed to finalize AXL library @ %s:%d",
-      __FILE__, __LINE__
-    );
+  if (!poststage_skip) {
+    int axl_rc = AXL_Finalize_comm(scr_comm_world);
+    if (axl_rc != AXL_SUCCESS) {
+      scr_abort(-1, "Failed to finalize AXL library @ %s:%d",
+	__FILE__, __LINE__
+      );
+    }
```

Comment:
Does it break things if we leave this enabled?  I'm worried about introducing memory leaks if we don't call it.

Reply:
No, looks like I don't need that at all.  I'll remove